lib/outcome_checker.rb

```diff
@@ -0,0 +1,20 @@
+class OutcomeChecker
+  def win?(board)
+    winning_combo?(board.lines)
+  end
+
+  def draw?(board)
+    board.full? && !win?(board)
+  end
+
+  private
+
+  def winning_combo?(lines)
+    line_statuses = lines.map { |line| all_equal?(line) }
+    line_statuses.include?(true)
```

Comment:
```suggestion
    line_statuses.any?
```

Reply:
Or maybe just a one liner?
```suggestion
    lines.any? { |line| all_equal?(line) }
```

Reply:
Love it, thanks!